src/uptdd.f

```diff
@@ -1,62 +1,35 @@
-C>                           OPENED AT ONE TIME INCREASED FROM 10 TO 32
-C>                           (NECESSARY IN ORDER TO PROCESS MULTIPLE
-C>                           BUFR FILES UNDER THE MPI)
-C> 2003-11-04  J. ATOR    -- ADDED DOCUMENTATION
-C> 2003-11-04  S. BENDER  -- ADDED REMARKS/BUFRLIB ROUTINE
-C>                           INTERDEPENDENCIES
-C> 2003-11-04  D. KEYSER  -- UNIFIED/PORTABLE FOR WRF; ADDED HISTORY
-C>                           DOCUMENTATION; OUTPUTS MORE COMPLETE
-C>                           DIAGNOSTIC INFO WHEN ROUTINE TERMINATES
-C>                           ABNORMALLY
-C> 2014-12-10  J. ATOR    -- USE MODULES INSTEAD OF COMMON BLOCKS
+C> ### Program History Log
+C> | Date | Programmer | Comments |
+C> | -----|------------|----------|
+C> 1994-01-06 | J. Woollen  |  original author
+C> 1995-06-28 | J. Woollen  |  increased the size of internal bufr table arrays 
+C> 1998-07-08 | J. Woollen  |  replaced call to cray library routine "abort" with bufrlib routine "bort"
+C> 1999-11-18 | J. Woollen  |  the number of bufr files which can be opened at one time increased from 10 to 32
+C> 2003-11-04 | J. Ator     |  added documentation
+C> 2003-11-04 | S. Bender   |  added remarks/bufrlib routine interdependencies
+C> 2003-11-04 | D. Keyser   |  unified/portable for wrf; added history documentation; outputs more complete diagnostic info 
+C> 2014-12-10 | J. Ator     |  use modules instead of common blocks
 C>
-C> USAGE:    CALL UPTDD (ID, LUN, IENT, IRET)
-C>   INPUT ARGUMENT LIST:
-C>     ID       - INTEGER: POSITIONAL INDEX OF PARENT MNEMONIC WITHIN
-C>                INTERNAL BUFR TABLE D ARRAY TABD
-C>     LUN      - INTEGER: I/O STREAM INDEX INTO INTERNAL MEMORY ARRAYS
-C>     IENT     - INTEGER: ORDINAL INDICATOR OF CHILD MNEMONIC TO RETURN
-C>                FROM WITHIN TABD(ID,LUN) SEQUENCE:
-C>                       0 = return a count of the total number of child
-C>                           mnemonics within TABD(ID,LUN)
-C>
-C>   OUTPUT ARGUMENT LIST:
-C>     IRET     - INTEGER: RETURN VALUE (SEE REMARKS)
-C>
-C> REMARKS:
-C>    THE INTERPRETATION OF THE RETURN VALUE IRET DEPENDS UPON THE INPUT
-C>    VALUE IENT, AS FOLLOWS:
-C>
-C>    IF ( IENT = 0 ) THEN
-C>       IRET = a count of the total number of child mnemonics within
-C>              TABD(ID,LUN)
-C>    ELSE
-C>       IRET = the bit-wise representation of the FXY value
-C>              corresponding to the IENT'th child mnemonic of
-C>              TABD(ID,LUN)
-C>    END IF
+C> @author Woollen @date 1994-01-06
+
+C> This subroutine returns the bit-wise representation of the fxy value corresponding to, sequentially, 
+C> a particular (ient'th) "child" mnemonic of a table d sequence ("parent") mnemonic.
 C>
+C> @remark The interpretation of the return value iret depends upon the input value ient, as follows:
+C> ient=0 - iret = a count of the total number of child mnemonics within tabd(id,lun)
```

Comment:
Probably you want these two items to be bullet points. Try this:

```
C> @remark The interpretation of the return value iret depends upon the input value ient, as follows:
C> - ient=0 - iret = a count of the total number of child mnemonics within tabd(id,lun)
C> - ient>0 - iret = the bit-wise representation of the fxy value corresponding to the ient'th child mnemonic of tabd(id,lun)

```

Note the hyphens.